TipCatDotNet.Api/Models/HospitalityFacilities/Validators/IdentityClaimValidator.cs

```diff
@@ -0,0 +1,13 @@
+using FluentValidation;
+
+namespace TipCatDotNet.Api.Models.HospitalityFacilities.Validators
+{
+    public class IdentityClaimValidator : AbstractValidator<string?>
```

Comment:
Don't think we need this one. The check looks like a part of general application flow